packages/ember-toucan-core/rollup.config.mjs

```diff
@@ -26,11 +27,15 @@ export default {
     // not everything in publicEntrypoints necessarily needs to go here.
     addon.appReexports(['components/**/*.js']),
 
+    // compile <template> tag into plain JS
+    glimmerTemplateTag({ preprocessOnly: true }),
```

Comment:
Off the top of your head, any idea what is compiles to when there's a backing class? A `template` class method maybe?

Reply:
I'm honestly not sure but Preston would know!